tests/test_init.py

```diff
@@ -400,6 +400,85 @@ def test_import_replace_existing_with_column_renames(
             assert new_feature_tree == old_feature_tree
 
 
+def test_import_replace_ids(
+    data_archive,
+    tmp_path,
+    cli_runner,
+    chdir,
+):
+    with data_archive("gpkg-polygons") as data:
+        repo_path = tmp_path / "emptydir"
+        r = cli_runner.invoke(["init", repo_path])
+        assert r.exit_code == 0
+        with chdir(repo_path):
+            # initial import
+            r = cli_runner.invoke(
+                [
+                    "import",
+                    data / "nz-waca-adjustments.gpkg",
+                    "nz_waca_adjustments:mytable",
+                ]
+            )
+            assert r.exit_code == 0, r.stderr
+
+            # Now change some features:
+            with gpkg_engine(data / "nz-waca-adjustments.gpkg").connect() as conn:
+                conn.execute(
+                    """
+                    DELETE FROM "nz_waca_adjustments" WHERE id IN (4413497, 4411733)
+                    """
+                )
+                conn.execute(
+                    """
+                    UPDATE "nz_waca_adjustments" SET adjusted_nodes = adjusted_nodes + 5
+                    WHERE id IN (4408774, 4408145)
+                    """
+                )
+
+            r = cli_runner.invoke(
```

Comment:
comment on how the above IDs relate to what we're specifying below, and what we're checking?